README.md

```diff
@@ -1 +1,43 @@
 # salmon-sync
+
+This is a Github Action that syncs a folder to a Google Cloud bucket using `rclone`.
+This action is only meant to work for Deephaven's documentation. It could be used in a more general purpose way to sync a folder into any Google cloud bucket (with the proper credentials), but that is subject to change and may break in any version.
+
+## Parameters
+```
+inputs:
+  source:
+    required: true
+    type: string
+    description: 'The source directory to sync.'
+  destination:
+    required: true
+    type: string
+    description: 'The destination directory to sync. Relative to the bucket.'
```

Comment:
Does this have anything to do with the URL we will generate? Or does it need to include the git repository as the prefix? I see the example has `deephaven/deephaven.io/blog` so I'm curious what the importance of `deephaven/deephaven.io` is there

Reply:
I'm not entirely sure how the bucket is linked to the website, but this is just a location within the bucket. `deephaven/deephaven.io/blog` was just where Don wanted the blog content in this case.

Reply:
Don said the config on the salmon side would set up the link to the actual path for the docs site.

So it can be anything, but we should recommend using the git repo as the base so we have some hierarchy in the storage. I'd add "The GitHub repo organization and name should be the base" or something like that